### ISSUE TITLE
Tutorial uses model.eval() but there's no matching model.train()

### ISSUE THREAD

**Issue by juliangilbey** [pytorch/tutorials#1540]
The test function in this tutorial correctly calls `model.eval()`:

https://github.com/pytorch/tutorials/blob/aff18d84b936f62204167be2e8acedd8f8978c94/beginner_source/basics/quickstart_tutorial.py#L161-L173

But there is no corresponding `model.train()` to reset the model back into training mode. Since the `train` and `test` functions are called alternately later on, the training epochs after the first epoch will all be run with the model in eval mode.

It doesn't matter for this particular example, as there are no dropout or batchnorm layers or the like, but as a coding demonstration, it would be good to fix this.

**Comment by holly1238** [comment]
https://github.com/pytorch/tutorials/pull/1602
